src/actions/MapActions.js

```diff
@@ -24,7 +24,7 @@ export const updateViewStyle = (newViewStyle) => {
     };
 };
 
-export const mapLoaded = () => {
+export const mapHasLoaded = () => {
```

Comment:
`SearchScreen` was also using a property named `mapLoaded` and the conflicting names was causing issues. This name is more semantic anyway for an action.